Packs/QuestKace/Integrations/QuestKace/QuestKace.yml

```diff
@@ -0,0 +1,549 @@
+    - contextPath: QuestKace.Ticket.Status.ID
+      description: ID of the status of the ticket.
+      type: Number
+    - contextPath: QuestKace.Ticket.Status.Name
+      description: Name of the status of the tickets.
+      type: String
+    - contextPath: QuestKace.Ticket.Status.Ordinal
+      description: Ordinal of the status of the ticket.
+      type: Number
+    - contextPath: QuestKace.Ticket.Status.State
+      description: State of the status of the ticket.
+      type: String
+    - contextPath: QuestKace.Ticket.ID
+      description: ID of the ticket
+      type: Number
+    - contextPath: QuestKace.Ticket.Title
+      description: Title of the ticket.
+      type: String
+    - contextPath: QuestKace.Ticket.Summary
+      description: Summary of the ticket.
+      type: String
+    - contextPath: QuestKace.Ticket.Modified
+      description: Modified date of the ticket.
+      type: String
+    - contextPath: QuestKace.Ticket.Created
+      description: Created date of the ticket.
+      type: String
+    - contextPath: QuestKace.Ticket.HdQueueID
+      description: Queue number that the ticket is related to.
+      type: Number
+    - contextPath: QuestKace.Ticket.CcList
+      description: Cc list of the ticket.
+      type: String
+    - contextPath: QuestKace.Ticket.IsManualDueDate
+      description: 'Is manual due date '
+      type: Number
+    - contextPath: QuestKace.Ticket.Resolution
+      description: Resolution of the ticket.
+      type: String
+    - contextPath: QuestKace.Ticket.DueDate
+      description: Dua date of the ticket
+      type: String
+  - arguments:
+    - default: false
+      description: Title of the new ticket.
+      isArray: false
+      name: title
+      required: false
+      secret: false
+    - default: false
+      description: summary of the new ticket.
+      isArray: false
+      name: summary
+      required: false
+      secret: false
+    - default: false
+      description: name of the impact of the new ticket.
+      isArray: false
+      name: impact
+      required: false
+      secret: false
+    - default: false
+      description: category name of the new ticket.
+      isArray: false
+      name: category
+      required: false
+      secret: false
+    - default: false
+      description: status name fo the new ticket.
+      isArray: false
+      name: status
+      required: false
+      secret: false
+    - default: false
+      description: priority name of the new ticket.
+      isArray: false
+      name: priority
+      required: false
+      secret: false
+    - default: false
+      description: name of machine of the new ticket.
+      isArray: false
+      name: machine
+      required: false
+      secret: false
+    - default: false
+      description: name of the asset of the new ticket.
+      isArray: false
+      name: asset
+      required: false
+      secret: false
+    - default: false
+      description: Can enter custom fields for the new ticket.
```

Comment:
As we talked, let's make sure we have a more user friendly manner to use both OOTB fields and custom_fields.